### ISSUE TITLE
Chord format = Title case

### ISSUE THREAD

**Issue by davesyoung** [elsigh/chartcomposer#104]
even if I write in [g] or [bM] it should present as  "G" and "Bm"

<img width="910" alt="Screen Shot 2020-04-11 at 9 03 56 PM" src="https://user-images.githubusercontent.com/306901/79060288-2e003600-7c38-11ea-9973-1a5c60e41225.png">

**Comment by davesyoung** [comment]
you can do this in CSS

**Comment by elsigh** [comment]
Are there any chords that don't follow the pattern you are describing? I'm thinking you mean 
1) capitalize the first character
2) lower-case everything thereafter

**Comment by davesyoung** [comment]
CMaj?

**Comment by davesyoung** [comment]
Given the limited number of chords it probably makes sense to just have a recognized casing. If that is possible.

**Comment by elsigh** [comment]
Yeah, unfortunately too CMaj = C and is more readable as such.

**Comment by davesyoung** [comment]
Oh, yeah "C" is way better than "CMaj" so maybe it isn't that big of a deal since we wouldn't run into that problem unless someone manually typed in "CMaj"

Anyway this is a lower priority issue, so we can chat about it later. But basically I wanted to save folks the trouble of having to mess with uppercase and lowercase on phones.

**Comment by davesyoung** [comment]
How can i set this to be a P2 or P3?

**Comment by davesyoung** [comment]
@elsigh I just added P0-P3 "labels" not sure if you prefer a different method...

![image](https://user-images.githubusercontent.com/306901/79381982-a9533780-7f17-11ea-9dff-71a3df7a2f5b.png)

**Comment by elsigh** [comment]
Yeah, we can use PN instead of Low/Medium/High - doesn't matter too much to me. 

I don't know what there's a great solution here worth exploring beyond one where we build a giant map of strings->chords to do things like "CMaj" in chordpro in any case -> uppercase "C" in the Viewer.  We just need to write out the map.
